packages/components/src/Datalist/Datalist.scss

```diff
@@ -4,61 +4,55 @@ $tc-datalist-items-shadow-color: $shadow !default;
 $tc-typeahead-input-width: 260px !default;
 $tc-datalist-item-height: 40px !default;
 
-.tc-datalist {
-	display: flex;
-	align-items: center;
+
+.tc-datalist-form {
```

Comment:
Where is it used ? And if used, shouldn't it be declared in the form/widget scope instead ?